packages/palette-docs/content/docs/elements/layout/BorderBox.mdx

```diff
@@ -14,6 +14,6 @@ name: BorderBox
   <>
     <BorderBox mb={2}>One box...</BorderBox>
     <BorderBox mb={2}>Pushes down another</BorderBox>
-    <BorderBox mb={2} borderColor="red100">And another with border color</BorderBox> 
+    <BorderBox mb={2} p={1} borderColor="red100">And another with border color and tight padding</BorderBox> 
```

Comment:
Could you just see how setting a default `p: 2` interacts if you utilize px/py in a story?